### DISCUSSION TITLE
Background Dim and Story Board

### DISCUSSION THREAD

**Discussion by Yasakikondo74** [ppy/osu#30358] [Ideas]
I think this is a good idea for future creative expression, despite it being controversial to many players.
This would likely change how the competitive side of Osu, but it could help bring new players into the game.
This thought first came to my mind when I first watched BTMC video about Geometry Dash.
So my Ideas are:
Limit the background dim from 0% to 80%
- I know this would cause a lot of distraction and cause issue for players who get distracted easily while farming for PP.
- Though I think this would encourage beatmap creators to express themselves in designing storyboard and let players see the beauty in the beatmaps. I don't care if it's the most ugly or questionable designs. I still love them all.
- I also know that plain images are fine on their own but some songs would be a better fit with a storyboard or at least a background video would be fantastic.
Edit: if there's any lag issue related question, just allow the players to change from story board to plain image only.

**Comment by mixalisnew** [comment]
I see your point but i have to disagree. 100 dim has become something all players love, also im Geometry Dash there are settings to lower the dails of the map and some maps even have custom buttons to turn off effects witch most players use

**Comment by Joehuu** [comment]
Somewhere in 2013, this was a thing (limited to 60% dim and forced storyboard until pass): https://youtu.be/yZa2RQoKzpM?t=1522

Dunno of the reason why it was removed (probably community feedback).

**Reply by peppy** [reply]
I really liked this. I would love to bring it back.
